### DISCUSSION TITLE
Rendering LaTeX in tables and lists

### DISCUSSION THREAD

**Discussion by UlyssesZh** [community/community#17170] [Other Feature Feedback, Questions, & Ideas]
Refer to [this discussion](https://github.community/t/latex-math-rendering-in-markdown-lists-not-wokring/251780). LaTeX codes enclosed by dollar signs are not rendered in tables and lists.

```md
| Table | Example |
|-------|---------|
|       | $x$     |
```

| Table | Example |
|-------|---------|
|       | $x$     |

```md
- $x$
```
- $x$

```md
1. $x$
```
1. $x$

**Comment by peon374** [comment]
I would also love to see this work properly. Is this a bug or an _undocumented feature_ ?
